Tools/Scripts/webkitdirs.pm

```diff
@@ -3011,7 +3012,7 @@ Options specific to macOS:
   --locale=LOCALE                   Use a specific locale instead of the system region.
 
 Options specific to iOS:
-  --iphone-simulator                Run the app in an iPhone Simulator
+  --ios-simulator                   Run the app in an iPhone Simulator
```

Comment:
To add on to @aproskuryakov's comment here, this is case where the documentation reflects what we _should_ be doing, but the code in the script is actually wrong. Looking at `Tools/Scripts/webkitdirs.pm`, the good news is that a naive find-and-replace of `--ios-simulator` -> `--iphone-simulator` in `Tools/Scripts/webkitdirs.pm` will get the desired behavior, and that's what we should do here.

Reply:
ok, I will revert here and update the webkitdirs.pm instead. thank you for your inspecting:)

Reply:
@JonWBedard already updated, ptal